test/src/test/java/org/axonframework/test/aggregate/FixtureTest_Deadlines.java

```diff
@@ -44,61 +50,68 @@ void setUp() {
     @Test
     void testDeadlineScheduling() {
         fixture.givenNoPriorActivity()
-               .when(new CreateMyAggregateCommand("id"))
+               .when(CREATE_COMMAND)
                .expectScheduledDeadline(Duration.ofMinutes(TRIGGER_DURATION_MINUTES), "deadlineDetails");
     }
 
     @Test
     void testDeadlineSchedulingTypeMatching() {
         fixture.givenNoPriorActivity()
-               .when(new CreateMyAggregateCommand("id"))
+               .when(CREATE_COMMAND)
                .expectScheduledDeadlineOfType(Duration.ofMinutes(TRIGGER_DURATION_MINUTES), String.class);
     }
 
+    @Test
+    void testDeadlineSchedulingNameMatching() {
+        fixture.given(new MyAggregateCreatedEvent(AGGREGATE_ID, "deadlineName", "deadlineId"))
+               .when(new SetPayloadlessDeadlineCommand(AGGREGATE_ID))
+               .expectScheduledDeadlineWithName(Duration.ofMinutes(TRIGGER_DURATION_MINUTES), "payloadless-deadline");
```

Comment:
Tests for the `Instant scheduledTime` option are needed as well?

Reply:
Good argument! Something I was initially thinking about as well, but decided against because the `DeadlineManagerValidator` class (used under the hood to actually perform the `expectScheduledDeadline...` methods) calls the `DeadlineManagerValidator#assertScheduledDeadlineMatching(Instant, Matcher<?>)` in the `DeadlineManagerValidator#assertScheduledDeadlineMatching(Duration, Matcher<?>)` method.

Thus, tests which validate the expect methods using `Duration`, will always end up doing a validation with `Instant`.
I am assuming this was @m1l4n54v1c initial take on only providing tests for the expect methods using `Duration` too, hence why I stuck with it too.

Fine with adding the other set though, shouldn't be that big a deal.
Or, maybe @m1l4n54v1c has a changed opinion on this part at the moment, justifying tests for the expect-with-instant methods too.